cmd/postgres_replicator/README.md

```diff
@@ -0,0 +1,66 @@
+# Postgres replicator
+
+This tool replicates streaming data from BigQuery into PostgreSQL.
+
+
+## Configuration
+
+It is configured with a YAML file. Here is an example configuration file:
+
+```
+# Source database settings
+bigQuery:
+  projectID: ${BQ_PROJECT_ID}
+
+# Destination database settings
+postgres:
+  dbHost: 172.17.0.1
+  dbPort: 5432
+  dbUser: ${PG_USER}
+  dbPass: ${PG_PASS}
```

Comment:
can we use https://github.com/GoogleCloudPlatform/cloudsql-proxy?

It looks like you can run cloudsql-proxy locally, have it authenticate to cloudsql instance using application default credentials (https://cloud.google.com/sql/docs/mysql/connect-admin-proxy#authentication-options) and then simply connect to the local cloudsql-proxy endpoint?

Reply:
These credentials are used in addition to IAM permissions. I think the cloud proxy is intended to be used when you need to connect to the database locally (outside of some compute environment in GCP). When running on App Engine, IAM authentication is done automatically. (“By default, your app will authorize your connections using an App Engine service account.” https://cloud.google.com/sql/docs/postgres/connect-app-engine-flexible).